.github/dependabot.yml

```diff
@@ -16,6 +16,7 @@ updates:
       # Must be kept in-sync with libbpf
       - dependency-name: github.com/aquasecurity/libbpfgo
       # Forked/replaced dependencies
+      - dependency-name: github.com/alecthomas/kingpin/v2
```

Comment:
Please add the replace for all appropriate paths, it looks like we are replacing more than just the base go.mod.

(Same for master and other release branches.)

Reply:
Ah I see, I'll create a PR for master and then cherry-pick it in the existing PRs.

Reply:
Added, @codingllama

Reply:
Right, it looks like recent Dependabot changes by @jentfoo weren't backported. I don't think the bot runs on backports, so this may be moot in branches other than master. Sorry for the noise.

Reply:
Ah ok, but it doesn't hurt us to have these additional entires, so I will merge it in this form.